kubernetes-manifests/contacts.yaml

```diff
@@ -38,6 +38,8 @@ spec:
           value: "v0.2.0"
         - name: PORT
           value: "8080"
+        - name: LOG_LEVEL
```

Comment:
@WilliamG240  worth discussing what we want the default global vs. service log level to be.  Right now in your PR Sunny, contacts uses a global `info` and a service-level` info` -- Will, balancereader in your PR uses a global `info` and service-level `error`. Ideally all the Deployments files in `kubernetes-manifests/` should look the same -- I might actually argue that by default, *no* service has a service-level override -- the log level is commented out in the deployment, and all services use the global var.  Otherwise every service is overriding the global configmap.

Reply:
>  I might actually argue that by default, no service has a service-level override -- the log level is commented out in the deployment, and all services use the global var. Otherwise every service is overriding the global configmap.

This is what I was going to suggest. I think we should leave service-level variables commented out on master

We also need to document the new environment variable in the service's README

Reply:
For some reason the contacts service [README](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/master/src/contacts/README.md) already mentions the LOG_LEVEL env var, though I don't think it was used anywhere before?

Reply:
> > When I switched the log level to `debug` for contacts, I saw that the DB logs may not be formatted correctly --
> > ```
> > contacts-68f845c8c4-pj6gk contacts FROM contacts
> > contacts-68f845c8c4-pj6gk contacts WHERE contacts.username = %(username_1)s
> > contacts-68f845c8c4-pj6gk contacts 2020-06-15 12:01:19 | [DEBUG] | get_contacts | RESULT: <sqlalchemy.engine.result.ResultProxy object at 0x7f1bd392aa50>
> > ```
> > 
> > 
> > not sure if `%(username_1)s` is formatted right? should we be logging the actual username here?
> 
> I _think_ this has to do with sqlalchemy's stringifying of queries rather than the logger. Based on [this](https://docs.sqlalchemy.org/en/13/faq/sqlexpressions.html#rendering-bound-parameters-inline) it seems like we could change this format, but it introduces some security issues and isn't recommended. Should I still change this?

nope you can keep it as is!